node_ops/task_manager_module.cc

```diff
@@ -81,7 +81,7 @@ future<std::optional<tasks::task_status>> node_ops_virtual_task::get_status_help
         .kind = tasks::task_kind::cluster,
         .scope = "cluster",
         .state = get_state(entry),
-        .is_abortable = co_await is_abortable(),
+        .is_abortable = co_await is_abortable({}),
```

Comment:
It is hard to know what the {} is for. We can make is_abortable() get a default empty task hint if nothing is passed.